samples/nrf9160/serial_lte_modem/src/slm_util.c

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+#include <stdlib.h>
+#include <string.h>
+#include <errno.h>
+#include <stdio.h>
+#include "slm_util.h"
+
+#define PRINTABLE_ASCII(ch) (ch > 0x1f && ch < 0x7f)
+
+/**
+ * @brief Compare string ignoring case
+ */
+bool slm_util_cmp(const char *str1, const char *str2, u16_t str2_len)
+{
+	int i;
+
+	if (strlen(str1) != str2_len) {
+		return false;
+	}
+
+	for (i = 0; i < str2_len; i++) {
+		if (tolower(*(str1 + i)) != *(str2 + i)) {
+			return false;
+		}
+	}
+
+	return true;
+}
+
+
+/**
+ * @brief Compare name of AT command ignoring case
+ */
+bool slm_util_cmd_cmp(const char *cmd, const char *slm_cmd, u8_t slm_cmd_len)
```

Comment:
Again, do we need `slm_cmd_len`? And again I suggest `slm_util_cmd_casecmp`

Reply:
Renamed as suggested